health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -159,12 +160,15 @@ const Hypothesis = ({ category, assumptions: initialAssumptions, setShowToast, a
             <LabelFieldPair className="mp-hypothesis-label-field" style={{ alignItems: "center" }} key={index}>
               <div className="assumption-label">
-                <span>
+                <span className="assumption-label-icon-wrapper">
                   {`${t(item)}`}
                   {/* {category === "CAMPAIGN_VEHICLES" || sourceCheck === "CUSTOM" ? null : <span className="mandatory-span">*</span>} */}
                   {category === "CAMPAIGN_VEHICLES" || sourceCheck === "CUSTOM" ? null : (
                     <span className="icon-wrapper">
-                      <TooltipWrapper content={t(`HYPOTHESIS_MESSAGE_FOR_${item}`)} children={<InfoBannerIcon fill={"#C84C0E"} />} />
+                      <TooltipWrapper
+                        content={t(`HYPOTHESIS_MESSAGE_FOR_${item}`)}
+                        children={<InfoOutline fill={"#C84C0E"} width={"20px"} height={"20px"} />}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)